src/error.rs

```diff
@@ -92,16 +92,7 @@ impl core::fmt::Display for Unspecified {
 }
 
 #[cfg(feature = "std")]
-impl std::error::Error for Unspecified {
-    #[inline]
-    fn cause(&self) -> Option<&dyn std::error::Error> {
-        None
-    }
-
-    fn description(&self) -> &str {
-        Self::description_()
```

Comment:
`description_()` should be removed and inlined into the `Display` implementation.